cdap-docs/admin-manual/source/_includes/installation/configuration.txt

```diff
@@ -0,0 +1,284 @@
+ 
+   - If you want to use **a different HDFS user** than ``yarn``:
+ 
+     1. Check that there is |---| and create if necessary |---| a corresponding user on all machines
+        in the cluster on which YARN is running (typically, all of the machines).
+     #. Create an ``hdfs.user`` property for that user in ``conf/cdap-site.xml``::
+ 
+          <property>
+            <name>hdfs.user</name>
+            <value>my_username</value>
+            <description>User for accessing HDFS</description>
+          </property>
+ 
+     #. Check that the HDFS user owns the HDFS directory described by ``hdfs.namespace`` on all machines.
+  
+   .. _distribution-configuration-explore-service:
+ 
+   - To use the **ad-hoc querying capabilities of CDAP,** enable the CDAP Explore Service in
+     ``conf/cdap-site.xml`` (by default, it is disabled)::
+ 
+       <property>
+         <name>cdap.explore.enabled</name>
+         <value>true</value>
+         <description>Enable Explore functionality</description>
+       </property>
+ 
+     This feature cannot be used unless the cluster has a correct version of Hive installed.
+     See the section on `Hadoop/HBase Environment <installation.html#hadoop-hbase-environment>`__.
+     To use this feature on secure Hadoop clusters, please see these instructions on
+     `configuring secure Hadoop <configuration.html#secure-hadoop>`__.
+ 
+     **Note:** Some versions of Hive contain a bug that may prevent the CDAP Explore Service from starting
+     up. See `CDAP-1865 <https://issues.cask.co/browse/CDAP-1865>`__ for more information about the issue.
+     If the CDAP Explore Service fails to start and you see a ``javax.jdo.JDODataStoreException: Communications link failure``
+     in the log, try adding this property to the Hive ``hive-site.xml`` file::
+ 
+       <property>
+         <name>datanucleus.connectionPoolingType</name>
+         <value>DBCP</value>
+       </property>
+ 
+   - If you'd like to publish metadata updates to an external Apache Kafka instance, 
+     CDAP has the capability of publishing notifications upon metadata updates. Details on
+     the configuration settings and an example output are shown in the :ref:`Metadata and
+     Lineage section <metadata-update-notifications>` of the Developers' Manual.
+
+.. highlight:: console
+
+.. _distribution-configuration-for-secure-hadoop:
+
+Secure Hadoop
+-------------
+When running CDAP on top of a secure Hadoop cluster (using Kerberos
+authentication), the CDAP processes will need to obtain Kerberos credentials in order to
+authenticate with Hadoop, HBase, ZooKeeper, and (optionally) Hive.  In this case, the setting for
+``hdfs.user`` in ``cdap-site.xml`` will be ignored and the CDAP processes will be identified by the
+default authenticated Kerberos principal.
+
+**Note:** CDAP support for secure Hadoop clusters is limited to CDH 5.0.0 through CDH 5.4.5,
+and HDP 2.0 through 2.2.
```

Comment:
This statement is always going to be outdated, especially if we state the CDH patch release.  We support kerberos for the latest of HDP and CDH (2.3.x.x and 5.4.8), and Apache (bigtop) as well.  Is the purpose of this statement just to exclude MapR? if so, that's what it should say.